src/MCPClient/tests/test_sanitize.py

```diff
@@ -375,9 +361,6 @@ def test_sanitize_transfer_with_multiple_files(
             not in file_obj.currentlocation
         )
         assert "bulk-file" in file_obj.currentlocation
-        # Test the event details were written correctly for our object.
-        event = Event.objects.get(file_uuid=file_obj.uuid, event_type="name cleanup")
-        verify_event_details(event)
```

Comment:
Why does this need to be removed?

Reply:
The problem I had with `verify_event_details` is that it tests after fixtures like [microservice_agents.json](https://github.com/artefactual/archivematica/blob/stable/1.12.x/src/MCPClient/tests/fixtures/microservice_agents/microservice_agents.json) used in other tests. After the isolation work that stopped working. I'm going to add the following:

```python
from pytest_django.asserts import assertQuerysetEqual

assertQuerysetEqual(
    Event.objects.get(file_uuid=file_obj.uuid, event_type="name cleanup").agents.all(),
    [
        "<Agent: software; preservation system: %s; Archivematica>" % get_preservation_system_identifier(),
        "<Agent: organization; repository code: ORG; Your Organization Name Here>",
        "<Agent: Archivematica user; Archivematica user pk: 1; username=\"\", first_name=\"\", last_name=\"\">",
    ],
    transform=repr,
    ordered=False,
)
```

Reply:
@sevein Can't remember if you already mentioned this, but I just noticed this new block also fails when you run the tests multiple times and the database is reused.